### PR TITLE
added padding to standard accordion to be in-keeping with FAQ

### PR DESCRIPTION
**Pull request by tomatau**
Added some padding and variables to manage the padding and dimensions for the standard accordion.

Title colour is still TBC from the design as it might need to change when opened and closed.

Note: The margin bottom -10px hack is needed to stop padding being accumulated for closed FAQ items.

### CODE REVIEW

**Comment by alexflorisca** [comment]
Can you rebase and fix the heroku deploy so we can see it? Not sure what's wrong with it. works on some PRs but not others

**Comment by alexflorisca** [comment]
Also where are those $padding- vars defined?

**Comment by NowTV-Assurance** [comment]
We've made the new changes on-top of master (everything changed!) and the variables are local to where they are used.

IE. The standard accordion has some variables (that's all we changed) and these variables have nothing to do with the extended accordion (for the time being). So we just kept the variables out of reach of other rules.

**Comment by peter-mouland** [comment]
I might be wrong, but shouldn't the `now-accordion__content` have left and right padding on the `extended` version too?

Shall we put the design up here sos we know what we are comparing against?

**Comment by tomatau** [comment]
Adding the designs sounds like an ok idea but means more to maintain -- the plan was to keep the styleguide as the source of truth?

As for the extended, we didn't touch that in this PR, this was just the FAQ styles.

**Comment by peter-mouland** [comment]
sorry - when i said add the design, i just meant to the top of this PR so its easy to see what you're building against.

Maybe something was lost in the rebase.  You can see here (http://nowtv-styleguide.herokuapp.com/#accordion) that the extended content uses the same padding as the title

**Comment by tomatau** [comment]
Added that padding back in, good spot -- we did touch the extended to remove that little content slide on page load and the little jank it made when opening/closing.

faq design

File: src/components/accordion.scss [hunk 1]
```diff
@@ -87,8 +119,8 @@ a.now-accordion__title:hover{
 
 .now-accordion--transition {
   display: block;
-  transition: all 0.3s ease-in-out;
-  overflow: hidden
+  transition: max-height 0.3s ease-in-out;
```
Comment (peter-mouland) [inline src/components/accordion.scss]: i liked the padding-top and padding-bottom transition - looked good when opening and closing. I think it was the padding-left and right that was causing it to anim on page load?

**Comment by tomatau** [comment]
The transition still happens when opening and closing, just doesn't do a strange parallax with the content.

It was both top and left that was causing the anim on page load.

**Comment by alexflorisca** [comment]
Not sure what the exact pixel values should be for padding - not sure anyone will know - but :+1: on the basis that it looks ok in the review app.

**Comment by tomatau** [comment]
Those values came from Vick working on UI for assurance, he was very confident about them :)

**Comment by nowarkdev** [comment]
:+1:

@ldabiralai @nwc1991

**Comment by nowarkdev** [comment]
@benstokoe @alexjfno1  👍

**Pull request merged**